src/module/iotjs_module_i2c.cpp

```diff
@@ -49,6 +48,19 @@ I2c* I2c::GetInstance() {
 }
 
 
+void GetI2cArray(JObject* jarray, I2cReqData* req_data) {
+  // FIXME: Need to implement a function to get array info from JObject Array.
+  req_data->buf_len = jarray->GetProperty("length").GetNumber();
```

Comment:
For more safety, I'll follow your comment. I'll add assert for check there is length property or not.